graphite_api/functions.py

```diff
@@ -661,6 +661,43 @@ def asPercent(requestContext, seriesList, total=None):
     return resultList
 
 
+def divideSeriesLists(requestContext, dividendSeriesList, divisorSeriesList):
+    """
+    Iterates over a two lists and divides list1[0] by list2[0], list1[1] by list2[1] and so on.
+    The lists need to be the same length
+    """
+
+    if len(dividendSeriesList) != len(divisorSeriesList):
+        raise ValueError("dividendSeriesList and divisorSeriesList argument must have equal length")
+
+    results = []
+
+    #for dividendSeries in dividendSeriesList:
+    for i in range(0, len(dividendSeriesList)):
+        dividendSeries = dividendSeriesList[i]
+        divisorSeries = divisorSeriesList[i]
```

Comment:
to avoid using an index: `for dividentSeries, divisorSeries in zip(dividendSeriesList, divisorSeriesList):`

Reply:
I fixed that in the new commit.